pyupgrade/_plugins/imports.py

```diff
@@ -233,10 +233,11 @@ def _for_version(
         Mapping[tuple[str, str], str],
         Mapping[str, str],
 ]:
-    removals = {}
+    removals: dict[str, set[str]] = {}
     for ver, ver_removals in REMOVALS.items():
         if ver <= version:
-            removals.update(ver_removals)
+            for base, names in ver_removals.items():
+                removals.setdefault(base, set()).update(names)
```

Comment:
`collections.defaultdict(set)` would be better

Reply:
Done.
Initially I decided to avoid it, so it won't introduce any surprises when accessing missing keys and if there was any logic aronud it, but it seems there should be no issues with that.